3style-text/styles.css

```diff
@@ -0,0 +1,89 @@
+/* ---font styles--- */
+
+body {
+  font-family: "Public Sans", Verdana, sans-serif;
+  font-size: 18px;
+}
+
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: "Quicksand", sans-serif;
+  font-weight: 600;
+}
+
+
+/* ---font sizes--- */
+
+h1 {
+	font-size: 250%; /* 45px */
+}
+
+h2 {
+	font-size: 200%; /* 36px */
+}
+
+h3 {
+	font-size: 177.78%; /* 32px */
+}
+
+h4 {
+	font-size: 162.5%; /* 26px */
+}
+
+h5 {
+	font-size: 122%; /* 22px */
+}
+
+h6 {
+	font-size: 100%; /* 18px */
+}
+
+h3 {
+	font-weight: 700;
+}
+
+/* ---font weights--- */
+
+strong em,
+em strong {
+	font-family: "Quicksand", sans-serif;
+    font-style: normal;
+}
+
+/* ---font colors--- */
```

Comment:
Feels like there's some repetition in the following section / some previous values being overwritten?

Reply:
I think it's because as we go through the tutorial we do overwrite some values. Here we are seeing the benefits of different ways to make fonts responsive. Percent works, em is better, rem is best because it's based on the page default font value so you won't have to go in and change a bunch of values all over the place as you would with em!